avocado/utils/asset.py

```diff
@@ -411,6 +411,48 @@ def get_metadata(self):
     def asset_name(self):
         return os.path.basename(self.parsed_name.path)
 
+    @classmethod
+    def fetch_asset_by_name(cls, name, cache_dirs, expire=None,
+                            asset_hash=None):
+        """This method will return a cached asset based on name if exists.
```

Comment:
So, this method is not fetching the asset, but finding it in the cache. In this case, the method name is confusing. My suggestion is to use `find_asset_by_name` in this case.

Reply:
Makes sense, but I prefer `get_asset_by_name` because of the return.

Reply:
That is also fine for me.